code/modules/bs_snare/snare.dm

```diff
@@ -0,0 +1,41 @@
+/obj/item/clothing/accessory/bs_silk
+	name = "bluespace snare"
+	desc = "A bluespace snare. Looking at the edges of this thing, you see a faint blue ripple and spatial distortion."
+	icon = 'icons/obj/rig_modules.dmi'
+	icon_state = "teleporter"
+	w_class = ITEM_SIZE_SMALL
+	origin_tech = list(TECH_BLUESPACE = 6)
+
+	//var/locking_access = access_rd
+	var/obj/machinery/bssilk_hub/silk_hub
+	var/silk_id = "" //using by snare controller to teleport user to controller's mark
+
+/*/obj/item/clothing/accessory/bs_silk/proc/toggle_lock()
+	removable = !removable*/
+
+/obj/item/clothing/accessory/bs_silk/attackby(obj/item/I, mob/user)
+	if(isMultitool(I))// || removable)
+		var/input_id = input("Enter new BS Snare ID", "Snare ID", silk_id)
```

Comment:
there can be two items with same id.

Reply:
@ITCups, feature, i forget about it, i need to delete in hub "linked_mob"-var, and finding all mobs with snares with same ID